lib/action_cable_client.rb

```diff
@@ -143,11 +153,13 @@ def pinged(&block)
   # @param [String] message - the websockt message object
   def handle_received_message(message)
     return if message.empty?
+
     json = JSON.parse(message)
 
     if is_ping?(json)
       _pinged_callback&.call(json)
     elsif is_welcome?(json)
+      subscribe
```

Comment:
is welcome received after initial connection to a socket, or to a channel?
We should probably do a follow up PR with some in-line documentation

Reply:
If you look only at the commit that fix this (ie. b1c5b68c93e793256c187e83c6c1ccfdd1627d47), you can see that I don't change the behavior in case you provide a block to `#connected` : it `#subscribe` then yield user block.
With this commit, even you don't provide block to `#connected`, `#subscribe` is executed.